frontend/mulighetsrommet-veileder-flate/src/components/joyride/utils.ts

```diff
@@ -0,0 +1,7 @@
+export const localeStrings = () => ({
```

Comment:
En grunn til at denne er en funksjon i stedet for et objekt?

Reply:
> In JavaScript, functions are called Function Objects because they are objects. Just like objects, functions have properties and methods, they can be stored in a variable or an array, and be passed as arguments to other functions.

https://www.dofactory.com/javascript/function-objects